templates/cloud/index.html

```diff
@@ -1,250 +1,364 @@
-      </ul>
-    </div>
-  </div>
-  <div class="u-fixed-width">
-    <p>Major public cloud providers include <a href="/aws">Amazon Web Services (AWS)</a>, <a href="/azure">Microsoft Azure</a>, <a href="/gcp">Google Cloud Platform</a>, Oracle and IBM Cloud.</p>
-    <p>
-      <a href="/download/cloud">Learn more about Ubuntu on Public Clouds&nbsp;&rsaquo;</a>
-    </p>
-  </div>
-</section>
+  <section class="p-section">
+    <div class="p-section--shallow">
+      <div class="row--50-50">
+        <hr class="p-rule" />
+        <div class="col">
+          <h2>What is a public cloud?</h2>
+        </div>
+        <div class="col">
+          <p>
+            In a <a href="/cloud/public-cloud">public cloud environment</a>, compute, storage and other infrastructure resources are provided as a service by an external provider.
+          </p>
+        </div>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-9 col-start-large-4 col-medium-5 col-start-medium-2">
+        <hr class="is-muted" />
+        <div class="row">
+          <div class="col-3 col-medium-2">
+            <h5>Advantages:</h5>
+          </div>
+          <div class="col-6 col-medium-3">
+            <ul class="p-list--divided">
+              <li class="p-list__item is-ticked">
+                Immediate access to infrastructure as a service (IaaS) can significantly accelerate time to market for organisations.
+              </li>
+              <li class="p-list__item is-ticked">
+                Public clouds provide the ability to scale resource consumption as needed and virtually without limits.
+              </li>
+            </ul>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-9 col-start-large-4 col-medium-5 col-start-medium-2">
+        <hr class="is-muted" />
+        <div class="row">
+          <div class="col-3 col-medium-2">
+            <h5>Disadvantages:</h5>
+          </div>
+          <div class="col-6 col-medium-3">
+            <ul class="p-list--divided">
+              <li class="p-list__item is-crossed">
+                <a href="https://go.451research.com/Emerging-pricing-model-for-private-cloud">Can be more expensive</a> than an efficiently-managed and well-utilised private cloud.
+              </li>
+              <li class="p-list__item is-crossed">
+                In addition to the compute costs, there can be unexpected ancillary costs like egress (taking your data out of the cloud)
+              </li>
+            </ul>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="row--50-50">
+      <hr class="is-muted" />
+      <div class="col"></div>
+      <div class="col">
+        <p>
+          <a href="/download/cloud">Learn more about Ubuntu on Public Clouds&nbsp;&rsaquo;</a>
+        </p>
+      </div>
```

Comment:
```suggestion
    <div class="row">
      <hr class="is-muted" />
      <div class="col-start-large-7">
        <p>
          <a href="/download/cloud">Learn more about Ubuntu on Public Clouds&nbsp;&rsaquo;</a>
        </p>
      </div>
```
It would be better to use the [empty columns](https://vanillaframework.io/docs/patterns/grid#empty-columns) pattern here for clarity